src/strategies/sushiswap/index.ts

```diff
@@ -76,5 +194,18 @@ export async function strategy(
         });
     });
   }
+  if (options.useStakedBalances === 'true') {
```

Comment:
Would be great if you can add a small description in README.md of this strategy, that can explain what this option can do